OpenEdXMobile/src/main/java/org/edx/mobile/model/api/TranscriptModel.java

```diff
@@ -56,6 +59,10 @@
             languageArray.put(context.getString(R.string.cc_spanish_code),
                     context.getString(R.string.lbl_cc_spanish));
         }
+        if (arabicUrl != null) {
+            languageArray.put(context.getString(R.string.lbl_cc_arabic_code),
+                    context.getString(R.string.lbl_cc_arabic_code));
```

Comment:
You should add `arabicUrl` comparison in the `equals()` method as well. Otherwise looks good!